core/thread/id.go

```diff
@@ -353,6 +354,7 @@ type Info struct {
 }
 
 // GetOwnLog returns the first log found with a private key.
+// This is a strict owership check, vs returning all directly 'managed' logs.
```

Comment:
Updated docs here, but leaving the behavior the same. Essentially, this is a more 'strict' check on ownership, where you _do_ need to the have private key locally.

Reply:
Maybe I'll see below, but is this still needed? Now that there _could_ be multiple private keys, this could start returning different things at different times.

Reply:
This is actually now ok, as we have the distinction between "owned" (have the private key) and "managed" (have only the public key)